src/Core/Header.php

```diff
@@ -72,11 +73,16 @@ class Header
      * @throws ParseException If unable to parse the config file
      * @return string
      */
-    public static function setupHeader($assets = [])
+    public static function setupHeader($assets = [], $echoOutput = true)
     {
         // Required tag
-        echo '<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />';
-        self::setupAssets($assets, true);
+        $output = '<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />';
+        $output .= self::setupAssets($assets, true);
+        if ($echoOutput == true) {
+            echo $output;
+        } else {
+            return $output;
+        }
     }
```

Comment:
Then just need to change the following line:
https://github.com/openemr/openemr/blob/081634938d424d99c83723fa2a59609c94aceee7/src/Core/Header.php#L105
to:
```
return self::includeAsset($assets);
```